distribution/lib/Standard/Test/0.0.0-dev/src/Extensions.enso

```diff
@@ -70,23 +72,24 @@ Error.should_fail_with self matcher frames_to_skip=0 unwrap_errors=True =
 
          example_should_equal = Examples.add_1_to 1 . should_equal 2
 Any.should_equal : Any -> Integer -> Spec_Result
-Any.should_equal self that frames_to_skip=0 = case self == that of
-    True -> Spec_Result.Success
-    False ->
-        loc = Meta.get_source_location 2+frames_to_skip
-        additional_comment = case self of
-            _ : Vector -> case that of
-                _ : Vector ->
-                    case self.length == that.length of
-                        True ->
-                            diff = self.zip that . index_of p->
-                                p.first != p.second
-                            "; first difference at index " + diff.to_text + " "
-                        False -> "; lengths differ (" + self.length.to_text + " != " + that.length.to_text + ") "
+Any.should_equal self that frames_to_skip=0 = rhs_error_check that <|
```

Comment:
Finally I see it. This wraps the original code in `rhs_error_check` helper function. It doesn't change API of `Any.should_equal` (it is not easy to see that when argument definition is mixed with code on the same line). As a result of the additional function call, the frames shift and thus `4+frame_to_skip`. The rest remains unchanged.